src/components/models/Block2Feedbacks.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Grid, Typography } from '@material-ui/core';
+import { withStyles } from '@material-ui/styles';
+import { baseStyles, spacing, Dot } from '../styledComponents';
+
+const styles = {
+  ...baseStyles,
+  card: {
+    margin: `0px ${spacing(2)}`,
+    padding: spacing(4),
+    border: '1px solid #DCDCDC',
+    boxSizing: 'border-box',
+    boxShadow: '0px 30px 20px rgba(0, 0, 0, 0.05)',
+  },
+  body2: {
+    fontStyle: 'italic',
+    marginBottom: spacing(3),
+  },
+  caption: {
+    fontFamily: 'Montserrat',
+  },
+  photo: {
+    marginRight: spacing(2),
+  },
+  controllers: {
+    marginTop: spacing(5),
+  },
+};
+
+class Block2Feedbacks extends React.Component {
+  constructor(props) {
+    super(props);
+    const { comments, viewsToShow } = this.props;
+    this.state = { visibleComments: comments.slice(0, viewsToShow) };
+  }
+
+  getControllers() {
+    const { comments, viewsToShow } = this.props;
+    const { visibleComments } = this.state;
+    const items = [];
+    const numberOfDot = Math.ceil(comments.length / viewsToShow);
+    for (let indice = 0, currVisibleComments = null, currIndex = 0; indice < numberOfDot; indice += 1) {
```

Comment:
I think the word you're looking for here is `index` (not `indice`).

Reply:
Sorry FrenGlish :smile:

Reply:
Haha, pas de probleme! Si j'écrivais en Français, ce serait encore pire!